src/applications/caregivers/components/SubmitError/DownloadLink.jsx

```diff
@@ -9,6 +9,7 @@ import { apiRequest } from 'platform/utilities/api';
 const DownLoadLink = ({ form }) => {
   const [PDFLink, setPDFLink] = useState(null);
   const formData = submitTransform(formConfig, form);
+  const veteranFullName = form.data.veteranFullName;
```

Comment:
I thought about using the `?.` optional ternary but decided not to since this is required data on the form and this error is at the end of the UX.